### PR TITLE
use node-nut instead of calling local upsc binary

### PR DESCRIPTION
**Pull request by emes**
thought this would be nicer for docker installs and generally not having to locally install NUT.  I'm also sending errors to node.status and updated for Node-RED 1.0.  Did a little refresh on the edit config dialog and help text.  Everything should be backwards compatible.

### CODE REVIEW

File: nut-ups/nut-ups.html [hunk 1]
```diff
@@ -8,32 +8,66 @@
...
+    <p>The ups@server info can be configured in the node or it can be set
+    by <code>msg.ups</code>.</p>
+	<p>Likewise the port value for the NUT server can be configured in the node or
+	set by <code>msg.port</code></p>
+
+<h3>References</h3>
+    <ul>
+        <li><a href='https://networkupstools.org'>Network UPS Tools</a> - NUT project</li>
+        <li><a href='https://github.com/skarcha/node-nut'>Node-NUT</a> - NodeJS module that implements a NUT client</li>
+        <li><a href='https://github.com/myfreeweb/node-red-contrib-nut-ups'>GitHub repository</a> - NodeJS module that implements a NUT client</li>
```
Comment (valpackett) [inline nut-ups/nut-ups.html]: probably should have a different description than `node-nut`

File: nut-ups/nut-ups.js [hunk 2]
```diff
@@ -1,39 +1,65 @@
...
+module.exports = function(RED) {
+    'use strict';
+    function NutUpsNode(config) {
+        RED.nodes.createNode(this, config);
+        const node = this;
+
+        const Nut = require('node-nut');
+
+        this.on('input', function(msg, send, done) {
+			send = send || function() { node.send.apply(node,arguments) }
```
Comment (valpackett) [inline nut-ups/nut-ups.js]: extra indent here

File: package.json [hunk 3]
```diff
@@ -1,14 +1,13 @@
 {
   "name": "node-red-contrib-nut-ups",
-  "version": "1.0.2",
+  "version": "1.1.0",
```
Comment (valpackett) [inline package.json]: please don't bump versions in PRs

**Comment by valpackett** [comment]
Cool, thanks! I'm not sure why you decided to change the code style though. My preference has been [standard](https://standardjs.com/rules.html)-but-with-tabs. I'll add an `.eslintrc` to make auto-formatting with `eslint --fix` work.

File: nut-ups/nut-ups.html (hunk 1 above)
Comment (emes) [reply nut-ups/nut-ups.html]: Didn't need to be in help text as it's a dev item, so I removed.

File: nut-ups/nut-ups.js (hunk 2 above)
Comment (emes) [reply nut-ups/nut-ups.js]: eslint fixed

File: package.json (hunk 3 above)
Comment (emes) [reply package.json]: whoops

**Comment by emes** [comment]
Let me know if anything else.

**Comment by valpackett** [comment]
All good, thanks. I'll release a new version on npm now.

**Pull request merged**